components/Article/Page.js

```diff
@@ -321,12 +320,10 @@ class ArticlePage extends Component {
           return (
             <Fragment>
               {!isFormat && <PayNote.Before />}
-              <SSRCachingBoundary cacheKey={article.id}>
-                {() => renderMdast({
-                  ...article.content,
-                  format: meta.format
-                }, schema)}
-              </SSRCachingBoundary>
+              {() => renderMdast({
```

Comment:
`SSRCachingBoundary` takes a function as a child for lazy computing (only on cache miss). When removing `SSRCachingBoundary` you need to also remove the anonymous function.

<img width="1535" alt="screen shot 2018-03-12 at 04 05 24" src="https://user-images.githubusercontent.com/410211/37263931-dba5015a-25aa-11e8-9c63-5f46b2d22653.png">